src/main/java/frc/robot/commands/DefaultTeleop.java

```diff
@@ -93,20 +93,24 @@ public void execute() {
         double yAxis = -driver.getRawAxis(translationSup);
         double xAxis = -driver.getRawAxis(strafeSup);
         double rotationAxis = driver.getRawAxis(rotationSup);
+        alignPose = AlignPosition.getAlignPose();
+
+        if(alignPose != null){
```

Comment:
We should also check that AlignPosition equals SpeakerPos, since that's really the only scenario where we want to calculate an angle based on our position.

Reply:
done